packages/tcplay-veracrypt/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/veracrypt/tc-play
+TERMUX_PKG_DESCRIPTION="Free and simple TrueCrypt implementation based on dm-crypt."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
```

Comment:
The license is BSD 2-Clause based on the license file in the git repo: https://github.com/veracrypt/tc-play/blob/master/LICENSE

Reply:
Changed